helpers/objectMappers.ts

```diff
@@ -209,7 +209,7 @@ export const convertToFeedData = (data: FeedQuery) => {
     } else if (obj.user_achievement && obj.user_achievement.achievement) {
       feedData.push(convertToAchievementFeedData(obj.user_achievement.achievement, obj.user, obj.created_at));
     } else {
-      console.error('Error converting feed data. Data is not an activity or achievement!');
+      console.log('This feed element does not have a linked activity or achievement.');
```

Comment:
There exists feed elements in the db which does not have a valid link to an activity or achievement, consequently throwing an error here. Could delete those items, but don't think it matters as much. However, keeping it as a `console.log` because I think might be useful information when we are testing the application. If you disagree we can remove it:)

Reply:
I agree